testing/darktable/APKBUILD

```diff
@@ -1,7 +1,7 @@
 # Contributor: Kevin Daudt <ops@ikke.info>
 # Maintainer: Kevin Daudt <ops@ikke.info>
 pkgname=darktable
-pkgver=2.2.0_rc1
+pkgver=2.2.0_rc2
```

Comment:
Looking at travis output, and  pkgs.alpinelinux.org, i think these can be added here:
```
libsecret-dev
openjpeg-dev
cups-dev
```

And `graphicsmagick-dev` is avaliable in community Repository